full.txt

```diff
@@ -32429,7 +32503,7 @@ zyjal
 zyjoke
 zyjypi
 zykovaekaterin
-zylnu
+zylnuacasiobirewall
```

Comment:
lines got combined, maybe try the script at https://github.com/steemit/redeemer-irredeemables/pull/46?

Reply:
Thank you for noticing. I updated it using the makefile in #46. Will that PR be merged soon?